### PR TITLE
Calendar, localstorage & daily reset

### PR DESCRIPTION
**Pull request by WE-dev1**

### CODE REVIEW

File: components/AddCalories/AddCalories.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import useCalorieStore from "../../utils/useCalorieStore";
 import { AddCalorieButton, AddCalorieForm, FixedInput } from "./styles";
 
-export default function AddCalories() {
+export default function AddCalories({ setIsFormVisible }) {
```
Comment (actyralf) [inline components/AddCalories/AddCalories.js:4]: It's best practice to prefix callback props with `on` - you could e.g. call it `onHide` or `onClose` as you only call it to close the overlay and not to show it.

File: components/Calendar/styles.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import { css } from "styled-components";
+import styled from "styled-components";
+
+export const StyledCalendarContainer = styled.div``;
+
+/* export const calendarStyles = css`
+  .calendar {
```
Comment (actyralf) [inline components/Calendar/styles.js:7]: Please remove unused code.

File: pages/index.js [hunk 3]
```diff
@@ -1,48 +1,104 @@
...
+    if (today !== currentDay) {
+      setDailyCount(0);
+      setCurrentDay(today);
+    }
+  }, 5000);
+  // <----------------------------------------
 
   useEffect(() => {
-    document.addEventListener("click", () => setIsListVisible(false));
+    document.addEventListener("click", () => {
```
Comment (actyralf) [inline pages/index.js:39]: You should remove your event listener on unmount by returning a cleanup function. 
[How to clean up event listeners](https://stackoverflow.com/questions/61444561/react-hooks-how-to-clean-event-listeners)

File: utils/useCalorieStore.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
 import { create } from "zustand";
+import { persist } from "zustand/middleware";
 
-const useCalorieStore = create((set) => ({
+/* const useCalorieStore = create((set) => ({
```
Comment (actyralf) [inline utils/useCalorieStore.js:4]: please remove unused code.

**Review by actyralf — CHANGES_REQUESTED** [review]
Great work! 
Some tiny remarks below...

File: components/AddCalories/AddCalories.js (hunk 1 above)
Comment (WE-dev1) [reply components/AddCalories/AddCalories.js:4]: I thought no changing the name would be a good Idea to increase readability

**Comment by CoderMarkus** [comment]
LGTM!

**Comment by CoderMarkus** [comment]
LGTM!

**Pull request merged**